### ISSUE TITLE
DISK does not create a workflow on wings.

### ISSUE THREAD

**Issue by mosoriob** [KnowledgeCaptureAndDiscovery/DISK-WEB#56]
**Describe the bug*

DISK does not create a workflow on wings.

**To Reproduce**
1. Go to (https://disk.isi.edu/hypotheses/Hypothesis-eyhHQoDP0qhP)
2. Click on Update

**Expected behavior**
Create the workflow

**Screenshots**
```bash
Quering hypothesis: Hypothesis-6f28AYbUooqn
[(http://localhost:8080/disk-server/admin/hypotheses/Hypothesis-6f28AYbUooqn#goal http://disk-project.org/ontology/hypothesis#associatedWith Temperature), (http://localhost:8080/disk-server/admin/hypotheses/Hypothesis-6f28AYbUooqn#goal http://disk-project.org/ontology/hypothesis#associatedWith Humidity), (http://localhost:8080/disk-server/admin/hypotheses/Hypothesis-6f28AYbUooqn#goal http://disk-project.org/ontology/hypothesis#associatedWith Wind)]
LOI LOI-mOFYR83dYZpf got 1 results
Warning: file https://raw.githubusercontent.com/mosoriob/bikes_rent/master/bikes-part1.csv does not contain hash on Bikes-rental
Checking TriggeredLOI-3yMYjp38Z21U (LOI-mOFYR83dYZpf)
TLOI TriggeredLOI-3yMYjp38Z21U will be trigger
[R] Running execution thread
[R] Executing bikes with:
[R] - measure_per_day: bikes-part1.csv (1)
[R] - variable2: Temperature
[R] - variable3: Temperature
[R] - variable1: Temperature
Error parsing:
Can not deserialize instance of java.util.ArrayList out of VALUE_STRING token
at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@4a8c0c09; line: 1, column: 708] (through reference chain: edu.isi.wings.portal.classes.util.TemplateBindings["parameterBindings"]->java.util.HashMap["https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable1"])
Error parsing:
Can not deserialize instance of java.util.ArrayList out of VALUE_STRING token
at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@1e1fcc2a; line: 1, column: 708] (through reference chain: edu.isi.wings.portal.classes.util.TemplateBindings["parameterBindings"]->java.util.HashMap["https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable1"])
Error parsing:
Can not deserialize instance of java.util.ArrayList out of VALUE_STRING token
at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@7f2c0799; line: 1, column: 708] (through reference chain: edu.isi.wings.portal.classes.util.TemplateBindings["parameterBindings"]->java.util.HashMap["https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable1"])
+ users/admin/bikes/plan/getExpansions
+ toPOST: {"templateId":"https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#bikes","componentBindings": {},"parameterTypes": {"https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable1":"http://www.w3.org/2001/XMLSchema#string","https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable3":"http://www.w3.org/2001/XMLSchema#string","https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable2":"http://www.w3.org/2001/XMLSchema#string"},"parameterBindings": {"https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable1":"Temperature","https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable3":"Temperature","https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#variable2":"Temperature"},"dataBindings": {"https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/workflows/bikes.owl#measure_per_day":["https://wings.dev.disk.isi.edu/wings-portal/export/users/admin/bikes/data/library.owl#bikes-part1.csv"]}}
[R] Error: Could not get run id
[E] Finished: Something when wrong.
```

**Additional context**

**Comment by mosoriob** [comment]
107e36f303a6dafd64b87b93996eb31d446147fc doesn't fix the problem

**Comment by mosoriob** [comment]
the logs say nothing

```bash
2022-07-22T19:46:57.708Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T19:47:06.712Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T21:53:07.850Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T21:53:13.399Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T21:53:33.249Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T21:53:57.506Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
2022-07-22T21:54:14.472Z | Quering hypothesis: Hypothesis-eyhHQoDP0qhP
```

**Comment by mosoriob** [comment]
DISK is not creating the files

**Comment by mosoriob** [comment]
Fixed by #57
